source/scenes/tutorialIsland.js

```diff
@@ -0,0 +1,19 @@
+import { CONSTANTS } from "../constants.js";
+import { Level } from "../level.js";
+
+export class TutorialIsland extends Level{
+    constructor() {
+        super({
+            key: CONSTANTS.SCENES.TUTORIALISLAND,
+            killQuest: 10,
+            background: { name: 'tutorialIsland', path: 'source/assets/Tutorial_Island_Mine.png' },
+            minimap: { name: 'tutorialIslandMap', path: 'source/assets/TutorialIslandMap.png'},
```

Comment:
we should prob start being consistent with naming assets -> _ or no _?

Reply:
I'm partial to no_ but idc as long as we're consistent

Reply:
Added a task on trello to deal with this